index.src.html

```diff
@@ -1436,23 +1433,21 @@
 ## Defining extensions ## {#extension-specification}
 
 A definition of an extension must specify, at minimum, an extension identifier and an extension client argument sent via the
-{{getAssertion()}} or {{makeCredential()}} call (see below). Additionally, extensions may specify additional values in
-{{ClientData}}, `authenticatorData` (in the case of signature extensions), or both.
+{{getAssertion()}} or {{makeCredential()}} call. Additionally, extensions may specify additional values in {{ClientData}}, 
+`authenticatorData` (in the case of signature extensions), or both. Finally, if the extension requires any authenticator 
+processing, it must also specify an authenticator argument to be sent via the <a>authenticatorGetAssertion</a> or 
+<a>authenticatorMakeCredential</a> call.
 
-Note: An extension that does not define additions to {{ClientData}} nor `authenticatorData` is possible, but should be avoided.
-    In such cases, the [RP] would have no indication whether the extension was supported or processed by the client and/or
-    authenticator.
+Note: Any extensions that do not define additions to {{ClientData}} nor `authenticatorData` must be defined in a way that allows
+    the [RP] to unambiguously infer, from the returned signature or attestation statement, whether the intent of the extension
+    was followed by the client and/or authenticator. See the [[#extension-authenticator-selection]] for an example.
```

Comment:
sam suggested on the 15-jun webauthn call: Extensions should add to client data in some way to indicate to the RP that the extension was honored/respected/processed.